app/views/presentations/_sign_up_link.html.erb

```diff
@@ -0,0 +1,5 @@
+<%=
+  if event&.upcoming?
+    link_to 'Sign up', new_presentation_path(prepopulated_event: event.id)
```

Comment:
So I have a question that is meant more for discussion: 

Does it ever make sense to create a presentation that is not associated to a specific event? 
Is there ever a time when Im trying to create a presentation that I dont already know specifically which event I want it to be associated with?